elastic/kibana
# Redirect endpoint compression

## Summary

Closes https://github.com/elastic/kibana/issues/109092
Closes https://github.com/elastic/kibana/issues/109098

Two main things the PR adds are:

- Redirect endpoint locator params can be compressed now.
- Locators now have `.getRedirectUrl()` method.

To test both of those working:

1. Start Kibana with examples `yarn start --run-examples`.
2. Go to locator example app.
3. Navigate using a redirect link.
  1. You should see the params are compressed.


### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
